### PR TITLE
Added PhotoLibraryUsageDescription in order to acces fotos on IOS

### LINKED ISSUE #463: Application crashes when trying uploading pictogram om IOS
**Describe the bug**
When running the application on an iPad, trying to upload an image from the gallery will crash the application.

**To Reproduce**
1. Go to one of the weekplanners
2. Click to add a new activity
3. Press 'Tilføj fra galleri"
4. Press 'Tryk for at vælge billede'
5. See error (Application crashes)

**Expected behavior**
That the application will access the gallery to choose a photo.

**Actual behavior**
The application crashes

**Environment (please complete the following information):**
- iOS iPad emulator

### PR DESCRIPTION
**Pull request by Simon97**
Fixes #463

### CODE REVIEW

**Review by mads-bo — COMMENTED** [review]
@aau-giraf/sw609f20 you have been assigned this. Here's your checklist.
Please use the following checklist. Feel free to check it off as you go along.

**Code Design**

- [ ] The code is in the right place? (Both in terms of folder structure and class structure)
- [ ] This code could not, to the best of my knowledge, have reused existing code
- [ ] The code is not over-engineered. Examples of over-engineering: Implemented behavior for future problems
- [ ] The code does it introduce functionality that is not necessary for solving the problem

**Code Readability and Maintainability**

- [ ] Names are meaningful and self-documenting
- [ ] It is understandable, by reading the code, what it does
- [ ] The code is simple and readable. i.e. it contains no "hack", or obscure solution
- [ ] All new functionality is covered by tests
- [ ] It is understandable, by reading the tests, what they do
- [ ] The tests covers sensible cases. Both happy paths and exception paths
- [ ] The tests cover the full functionality. i.e., The tests fails if some of the requested functionality is missing
- [ ] Error messages in the code are understandable
- [ ] Confusing sections of code are covered by comments

**Functionality**

For this part, open the app, and test:

- [ ] The features that the code claims to implement, are actually implemented
- [ ] The code does not look like it has bugs
- [ ] All the new screens are reachable through in-app navigation

**Points for Discussion and Reflection**

- [ ] UI messages, that the user might meet, are actually helpful
- [ ] There are no errors that are bad practice. E.g. hard-coding a test database connection
- [ ] There is well written code in this. Remember to tell the developer!

**Review by mads-bo — COMMENTED** [review]
@aau-giraf/sw610f20 you have been assigned this. Here's your checklist.
Please use the following checklist. Feel free to check it off as you go along.

**Code Design**

- [ ] The code is in the right place? (Both in terms of folder structure and class structure)
- [ ] This code could not, to the best of my knowledge, have reused existing code
- [ ] The code is not over-engineered. Examples of over-engineering: Implemented behavior for future problems
- [ ] The code does it introduce functionality that is not necessary for solving the problem

**Code Readability and Maintainability**

- [ ] Names are meaningful and self-documenting
- [ ] It is understandable, by reading the code, what it does
- [ ] The code is simple and readable. i.e. it contains no "hack", or obscure solution
- [ ] All new functionality is covered by tests
- [ ] It is understandable, by reading the tests, what they do
- [ ] The tests covers sensible cases. Both happy paths and exception paths
- [ ] The tests cover the full functionality. i.e., The tests fails if some of the requested functionality is missing
- [ ] Error messages in the code are understandable
- [ ] Confusing sections of code are covered by comments

**Functionality**

For this part, open the app, and test:

- [ ] The features that the code claims to implement, are actually implemented
- [ ] The code does not look like it has bugs
- [ ] All the new screens are reachable through in-app navigation

**Points for Discussion and Reflection**

- [ ] UI messages, that the user might meet, are actually helpful
- [ ] There are no errors that are bad practice. E.g. hard-coding a test database connection
- [ ] There is well written code in this. Remember to tell the developer!

**Review by mads-bo — DISMISSED** [review]

File: ios/Runner/Info.plist [hunk 1]
```diff
@@ -2,6 +2,8 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+    <key>NSPhotoLibraryUsageDescription</key>
+    <string>Appen kræver adgang til dit fotobibliotek</string>
```
Comment (frederikspang) [inline ios/Runner/Info.plist]: Vi skal fortælle _hvorfor_ vi bruger adgangen i denne key.
```suggestion
	<key>NSPhotoLibraryUsageDescription</key>
	<string>Appen kræver adgang til dit fotobibliotek for at oprette nye piktogrammer</string>
```

**Review by frederikspang — CHANGES_REQUESTED** [review]

**Review by nicolaibjerring — APPROVED** [review]

**Review by frederikspang — APPROVED** [review]
Skide godt!

**Pull request merged**